### PR TITLE
Fix/solve snapshots tests failing node 18

### PR DESCRIPTION
**Pull request by simotae14**
### What does it do?

It solves the problem that we have with node 18.13 running the snapshots test, the problem was the unicode space character

### Why is it needed?

Because we have the job related to the frontend unit tests in node 18 failing

### How to test it?

You can try to run locally the unit test in node 18.13

- Refs https://github.com/nodejs/node/issues/46123

### CODE REVIEW

**Review by innerdvations — APPROVED** [review]

File: packages/core/helper-plugin/lib/src/components/FilterListURLQuery/tests/index.test.js [hunk 1]
```diff
@@ -13,6 +13,36 @@ import { createMemoryHistory } from 'history';
 import qs from 'qs';
 import FilterListURLQuery from '../index';
 
+jest.mock('react-intl', () => {
+  const reactIntl = jest.requireActual('react-intl');
+  const intl = reactIntl.createIntl({
+    locale: 'en',
+    messages: {
```
Comment (gu-stav) [inline packages/core/helper-plugin/lib/src/components/FilterListURLQuery/tests/index.test.js:20]: Could you store these messages as a constant above please and then re-use them in the two cases they are used?

**Review by joshuaellis — APPROVED** [review]
Worth writing a bit of documentation in `/docs` under `testing/unit` about this? Otherwise people _will_ question why we're mocking `react-intl` :) 

Could be a second PR to get this one merged though.

**Comment by gu-stav** [comment]
@joshuaellis I think I'd prefer to have a FE (sync) discussion about this first. To avoid developers running into this we should find a more global solution to this IMO (e.g. by globally mocking `react-intl` or `Date`). However this is a good emergency solution for now to fix the tests.

What do you think?

**Comment by joshuaellis** [comment]
I think at minimum a code comment should be there with those mocks – plans change, priorities are edited, who's to say when the permanent solution will be found and implemented? And I'd rather have that safety net then loose it!

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**